### PR TITLE
Captcha added in Signup and Contact Us page and phone numbers are clickable

### LINKED ISSUE #332: Add captcha to Contact us page, make phone numbers clickable

### LINKED ISSUE #333: Add captcha to sign up page

### PR DESCRIPTION
**Pull request by Sshashank18**
This closes #332. Also closes #333.

### CODE REVIEW

**Comment by ukey54** [comment]
I have tested both capcha and Contacts; they both working on my localhost.

File: public_website/views.py [hunk 1]
```diff
@@ -749,24 +751,41 @@ def get(self, request):
...
+                send_mail(
+                    subject='[Contact] ' + form.cleaned_data.get('subject'),
+                    message='',
+                    html_message=form.cleaned_data.get('message') + '<br><br> Senders email: ' + form.cleaned_data.get('emailid'),
+                    from_email='"Sawaliram" <mail@sawaliram.org>',
+                    recipient_list=['mail.sawaliram@gmail.com'],
+                    fail_silently=False,
+                )
+
+                messages.success(request, 'Your message has been sent! We will get back to you shortly')
```
Comment (badrihippo) [inline public_website/views.py:783]: This line needs to be translated as well! It should be something like `messages.success(request, _('Your message has been sent! We will get back to you shortly')`. Similarly for a few other messages below, including the context setting.

(In general, if any string is in English, please mark it as translatable using the `_()` command: that way translators can identify and convert those strings into the regional language.

**Review by badrihippo — CHANGES_REQUESTED** [review]
Good job on the commit! One change is that you need to mark the translatable strings (see my earlier comment). Apart from that it's all looking good, so if you make that change we can merge it :+1:

File: public_website/views.py (hunk 1 above)
Comment (Sshashank18) [reply public_website/views.py]: I have resolved this issue

Comment (badrihippo) [reply public_website/views.py]: Okay, looking good!

**Review by badrihippo — APPROVED** [review]

**Pull request merged**